src/main/java/io/sprucehill/zalando/api/nativecart/service/AccessTokenService.java

```diff
@@ -0,0 +1,37 @@
+package io.sprucehill.zalando.api.nativecart.service;
+
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+
+import io.sprucehill.zalando.api.nativecart.model.TokenInfo;
+import io.sprucehill.zalando.api.service.AbstractService;
+
+/**
+ * 
+ * @author dipteewarudkar
+ *
+ */
+public class AccessTokenService extends AbstractService implements IAccessTokenService {
+
+	@Override
+	public TokenInfo read() throws Exception {
+		HttpPost request = postRequest("/access_token?realm=%2Fcustomers&grant_type=client_credentials&client_id="+clientId+"&client_secret="+clientCredential);
```

Comment:
Please use `NameValuePair` to create the correct body / request parameters instead of doing String concatenation